### DISCUSSION TITLE
[Script request] teddycloud

### DISCUSSION THREAD

**Discussion by StefanX13** [community-scripts/ProxmoxVE#89] [Request script] [answered]
### Application Name

teddycloud

### Website

https://github.com/toniebox-reverse-engineering/teddycloud

### Description

teddyCloud is an alternative server for your Toniebox, allowing you to host the cloud services locally. This gives you the control about which data is sent to the original manufacturer’s cloud and allows you to host your own figurine audio files on e.g. your [NAS or any other server](https://tonies-wiki.revvox.de/docs/tools/teddycloud/setup/supported-server-hardware). It provides an easy to use WebGui.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Goldi95** [comment]
Love that idea

**Comment by dsiebel** [comment, marked as answer]
PR for review: https://github.com/community-scripts/ProxmoxVE/pull/1064
